modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc

```diff
@@ -1,107 +1,117 @@
-    file_sd_configs:
-      - files:
-          - 'couchbase_sd_config_my-cluster.yml'    # <.>
-        refresh_interval: 15s
-----
+== Call the Discovery API
 
-<.> This is the name that Prometheus will assign to the process retrieving data from the target nodes.
-<.> The username the process uses to log into the cluster.
-<.> The password the process uses to log into the cluster.
-<.> The name of target file. 
-The process will use this to read the addresses of the target nodes.
+You have two options to use the discovery API to configure Prometheus to collect metrics from Couchbase Server: file-based, or HTTP service discovery. 
 
-The directory should contain the following files.
+To use the file-based method, you call the discovery API and save the results to a file. 
+Then you configure Prometheus to read this file to get the list of nodes. 
+With this method, you have to take additional steps to update the list if you change the nodes in your Couchbase Database.
 
-[source, text]
-----
-📂 ~ (home directory)
-+ 📂 prometheus
-    📄 prometheus.yml
-    📄 couchbase_sd_config_my-cluster.yml
+To use the HTTP service discovery method, you configure Prometheus to get the list directly from the discovery API endpoint. 
+When you choose this method, Prometheus's defaults to  periodically calling the API endpoint to see if there have been changes. 
 
-----
 
-NOTE: The cluster configuration file will have a different name to the one shown here.
+The following sections explain each of these methods.
+
+=== File-Based Configuration
 
-== Install Prometheus
+With file-based configuration, you directly call the API discovery endpoint using `curl` or `wget` and save the output to a file. 
+By default, the endpoint returns the list of all nodes in the cluster in JSON format. 
+Prometheus accepts the list in either JSON or YAML format. 
 
-Open a terminal window and run the following command to provision Prometheus:
+To use the file-based method:
 
+. Call the Discovery API endpoint `/prometheus_sd_config` and save the results to a file. 
+The following example calls the discovery API endpoint using the `prometheus` user on a Couchbase node named `node1` and saves the output to a file named `couchbase_nodes.json`:
++
 [source, console]
 ----
-docker run \
-    -p 9090:9090 \    # <.>
-    -v ~/prometheus:/etc/prometheus \    # <.>
-    prom/prometheus
+curl  -u prometheus:_password_ \
+      http://node01:8091/prometheus_sd_config \
+      > couchbase_nodes.json
 ----
-<.> The port through which the Prometheus server is accessed and takes the form: `-p{nbsp}host-port:container-port`.
+. Copy the file to your Prometheus host if you did not run `curl` on your Prometheus host. 
+. Once the files are in place on the Prometheus host, edit the `prometheus.yml` configuration file. 
+Add a new job to collect metrics from your Couchbase Database. 
+In this job, add a `file_sd_configs` key entry that points to the list of nodes you created earlier. 
```

Comment:
"points to the list of nodes" -> "points to the file containing the list of nodes returned via the call to /prometheus_sd_config".

Reply:
Good point. Reworded to state the value needs to be the path to the file.